eng/references.targets

```diff
@@ -13,21 +20,32 @@
     <_TargetFrameworkDirectories>$(MSBuildThisFileDirectory)</_TargetFrameworkDirectories>
     <_FullFrameworkReferenceAssemblyPaths>$(MSBuildThisFileDirectory)</_FullFrameworkReferenceAssemblyPaths>
```

Comment:
@ericstj do you know if `_FullFrameworkReferenceAssemblyPaths` and `_TargetFrameworkDirectories` are required for `NetCoreAppCurrent`?

Reply:
This predates the .NET Core SDK when we were using just common/PCL targets to build 1.x.  Those targets had baked in assumptions that every framework would have a machine-wide targeting pack.  I suspect it's not needed any more since the SDK now supports fully package based frameworks.